sam-styles/packages/branding/elements/_tags.scss

```diff
@@ -401,4 +401,46 @@
     line-height: 1.5;
     margin-left: 0.75rem;
     text-decoration: none;
-  }
\ No newline at end of file
+  }
+
+  .sds-tag-right-align {
+    position: absolute;
+    top: 12px;
+    right: 12px;
+    z-index: 10;
+  }
+  
+  .sds-status-tag {
+    @include u-font("sans", "3xs");
+    @include u-text("normal");
+    padding: 6px 10px !important;
+    border-radius: 99rem;
+    display: inline-flex; 
+    align-items: center;
+    justify-content: center;
+    border: 1px solid transparent;
+  
+    &--info {
+      @include u-bg("info-lighter");
+      @include u-border("info-light");
```

Comment:
Did you confirm with Shannon that they wanted to use info-light from USWDS? For the buttons, we went with a one-off tweak in the color, so the info-light button won't match this border color.

Reply:
the description in the story says that the color they want for the border is info-light
![Screen Shot 2024-12-12 at 4 53 42 PM](https://github.com/user-attachments/assets/b81eaffd-5164-4bcc-ba1f-15258a825dce)

Reply:
Yes but between making that story and merging #718 the designers chose to adjust what hex code was used for the buttons that you are using on those card. If you just use the USWDS color token, the buttons and status border won't match.

@shannoncates-gsa can you confirm whether the info and error statuses should use the USWDS info and error tokens or should they use the adjusted tokens that we used for the buttons?

Reply:
I confirmed with Shannon and fixed the colors.

Reply:
You've still got info-lighter which was also re-defined to E7F6F8, that intended to be left to the USWDS color?